src/test/optimizer/strategy/disjunction_to_union_rule_test.cpp

```diff
@@ -0,0 +1,219 @@
+#include "gtest/gtest.h"
+
+#include "logical_query_plan/union_node.hpp"
+#include "optimizer/strategy/disjunction_to_union_rule.hpp"
+#include "strategy_base_test.hpp"
+#include "testing_assert.hpp"
+
+namespace opossum {
+
+class DisjunctionToUnionRuleTest : public StrategyBaseTest {
+ public:
+  void SetUp() override {
+    node_a = MockNode::make(MockNode::ColumnDefinitions{{DataType::Int, "a"}, {DataType::Int, "b"}}, "a");
+    a_a = node_a->get_column("a");
+    a_b = node_a->get_column("b");
+
+    node_b = MockNode::make(MockNode::ColumnDefinitions{{DataType::Int, "a"}, {DataType::Int, "b"}}, "b");
+    b_a = node_b->get_column("a");
+    b_b = node_b->get_column("b");
+
+    node_c = MockNode::make(MockNode::ColumnDefinitions{{DataType::Int, "a"}}, "c");
+    c_a = node_c->get_column("a");
+
+    node_d = MockNode::make(MockNode::ColumnDefinitions{{DataType::Int, "a"}}, "d");
+    d_a = node_d->get_column("a");
+
+    node_e = MockNode::make(MockNode::ColumnDefinitions{{DataType::Int, "a"}}, "e");
+    e_a = node_e->get_column("a");
+
+    _rule = std::make_shared<DisjunctionToUnionRule>();
+  }
+
+  std::shared_ptr<DisjunctionToUnionRule> _rule;
+
+  std::shared_ptr<MockNode> node_a, node_b, node_c, node_d, node_e;
+  LQPColumnReference a_a, a_b, b_a, b_b, c_a, d_a, e_a;
+};
+
+TEST_F(DisjunctionToUnionRuleTest, TwoExistsToUnion) {
+  // SELECT * FROM a WHERE EXISTS (
```

Comment:
why are you testing this with EXISTS predicates? just makes the tests hard to read, what's wrong with a dead simple a > 5?

Reply:
This was a relict because I assumed that only the `SubqueryToJoinRule` would benefit from the `DisjunctionToUnionRuleTest`.